### PR TITLE
Fix ansible lint issues

### PR DESCRIPTION
**Pull request by bhundven**
Some of these changes actually fix ansible style issues, others tell ansible-lint to ignore the issues.

I didn't fix the remaining `[303] yum used in place of yum module`, as that is fixed in #173. (good job @histamineblkr)

### CODE REVIEW

**Comment by bhundven** [comment]
ansible-lint
https://docs.ansible.com/ansible-lint/index.html

Maybe also look at pre-commit:
https://pre-commit.com/

File: tasks/pkg-suse.yml [hunk 1]
```diff
@@ -1,16 +1,16 @@
 ---
-- block: # Work around due to SNI check for SLES11
```
Comment (albertvaka) [inline tasks/pkg-suse.yml:2]: Have you tested this change doesn't break SLES11?

Comment (bhundven) [reply tasks/pkg-suse.yml:2]: I will test it tonight.

Comment (dabcoder) [reply tasks/pkg-suse.yml:2]: @bhundven did you have a chance to run your test(s)?

Comment (bhundven) [reply tasks/pkg-suse.yml:2]: I have not. I don't have a suse setup to test on. Do you have any suggestions for testing this?

Comment (albertvaka) [reply tasks/pkg-suse.yml:2]: If you split the PR in two, we can merge everything else now and wait for this part until someone can test it.

Comment (dabcoder) [reply tasks/pkg-suse.yml:2]: @bhundven any feedback on Albert's last comment? Let us know if you don't have time to split this PR and we can take care of the Debian part for now...

**Comment by dabcoder** [comment]
Followed up in #200, but thanks for initiating the work @bhundven

**Pull request closed without merging**